### DISCUSSION TITLE
Joining PvP Battlegrounds With a 5 Player Party

### DISCUSSION THREAD

**Discussion by armchairrockstar1** [chromiecraft/chromiecraft#6102] [Suggestions]
I think the server allows for a 3 player party to queue for BGs. However, some of us would appreciate it if we were allowed to go in with a party of 5 players. This won't make or break the games but could allow a small group from a guild to enjoy PvP together. I think 3 players are too little for this, so I would suggest increasing it to 5.

**Comment by balleny** [comment]
the server does not allow to queue battlegrounds as group (no matter the size - for a long time since the queue is itself has plenty of bugs).

and, although you mean it is a discussion, it would actually be a bug report, because on retail this is indeed possible.

lastly, always use the search function before opening bug reports or discussions, many of these topics already have been raised
